### PR TITLE
refine docs of elementwise_op etc.

### LINKED ISSUE #9188: API doc problems of elementwise_add, elementwise_div, elementwise_sub, elementwise_mul, elementwise_max, elementwise_min, elementwise_pow
#### 单词拼写、遗漏和语法等问题未列出，修改时请注意
*请参考[API注释标准](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/dev/api_doc_std_cn.md)！*
------
源文件：https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/ops.py

- elementwise_add
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-add
  - 缺Examples
  - 格式问题（数据类型）
  - 介绍须完善（如 returns 等）
  - 显示问题
image

- elementwise_div
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-div
  - 问题同上

- elementwise_sub
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-sub
  - 问题同上

- elementwise_mul
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-mul
  - 问题同上
  - 另
image

- elementwise_max
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-max
  - 问题同上

- elementwise_min
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-min
  - 问题同上

- elementwise_pow
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#elementwise-pow
  - 问题同上

### PR DESCRIPTION
**Pull request by luotao1**
fix #9188
1. refine the format and docs of elementwise_xxx_op.
image
image

2. fix typo of sequence_first/last_step.

### CODE REVIEW

**Comment by luotao1** [comment]
@skylarch

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -1409,7 +1409,7 @@ def sequence_pool(input, pool_type):
 
 def sequence_first_step(input):
     """
-    This funciton get the first step of sequence.
+    This function get the first step of sequence.
```
Comment (gongweibao) [inline python/paddle/fluid/layers/nn.py]: get->gets

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -1442,7 +1442,7 @@ def sequence_first_step(input):
 
 def sequence_last_step(input):
     """
-    This funciton get the last step of sequence.
+    This function get the last step of sequence.
```
Comment (gongweibao) [inline python/paddle/fluid/layers/nn.py]: gets

File: paddle/fluid/operators/elementwise_op.h [hunk 3]
```diff
@@ -72,23 +72,24 @@ The equation is:
...
+1. The shape of $Y$ is the same with $X$.
+2. The shape of $Y$ is a continuous subsequence of $X$.
 
 For case 2:
 
-$Y$ will be broadcasted to match the shape of $X$ and axis should be
-set to index of the start dimension to broadcast $Y$ onto $X$.
+1. Broadcast $Y$ to match the shape of $X$, where $axis$ is the start dimension index 
+   for broadcasting $Y$ onto $X$. 
+2. If $axis$ is -1 (default), $axis = rank(X) - rank(Y)$.
```
Comment (gongweibao) [inline paddle/fluid/operators/elementwise_op.h]: Need an example when `axis=-1`.

Comment (luotao1) [reply paddle/fluid/operators/elementwise_op.h]: Done

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (luotao1) [reply python/paddle/fluid/layers/nn.py]: Done

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (luotao1) [reply python/paddle/fluid/layers/nn.py]: Done

**Comment by skylarch** [comment]
格式范例：https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/dev/src/fc.py
line 1: extra space before "elementwise_add";
line 2: no period after "Operator"
line 5: please make changes on the format of the clarification of the equation's variables according to the above example.
Missing intro to the function's arguments. For clarity and consistency, please modify the "Parameters" part to match the format of the example above.

**Comment by luotao1** [comment]
@skylarch 
line 1. 所有的由C++转过去的，都由空格，需要统一改。
其余已改。效果图：
image

**Pull request merged**